decidim-admin/app/controllers/decidim/admin/organization_controller.rb

```diff
@@ -30,10 +30,15 @@ def update
       end
 
       def users
+        user_entities(current_organization.users)
+      end
+
+      def user_entities(relation = nil)
+        relation ||= current_organization.user_entities
```

Comment:
💬 It took me a bit to understand this. What do you think about moving this logic into a private method and call it from both `users` and `user_entities` endpoints?

Reply:
You're totally right. Changed